README.md

```diff
@@ -1,16 +1,17 @@
-<p align="center">
+# Tiktok Ads Source dbt Package ([Docs](https://fivetran.github.io/dbt_tiktok_ads_source/))
+
+<p align="left">
     <a alt="License"
-        href="https://github.com/fivetran/dbt_ad_reporting/blob/main/LICENSE">
+        href="https://github.com/fivetran/dbt_tiktok_ads_source/blob/main/LICENSE">
```

Comment:
The sample provided in the issue was specifically related to TikTok Ads Source. For this package we will need to retain the Ad Reporting reference. Would you be able to change this edit back to `dbt_ad_reporting`.